inkcut: fix build

raboof opened the pull request:
###### Description of changes

Fixes #225907

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

raboof committed:
Revert "python3Packages.pythonocc-core: 7.6.2 -> 7.7.0"

This reverts commit c8f346cb9df3e5da47077712fd192bebdce38d94.

c8f346cb9df3e5da47077712fd192bebdce38d94 was part of a bulk
auto-update of python packages, but broke the package as it
needs to be updated in tandem with `opencascade-occt`.

Adding `skipBulkUpdate` in a separate PR, #226020 .

raboof committed:
python3Packages.enamlx: fix build

bjornfor commented:
* Please keep each commit to the point. (I'm thinking of the `python310Packages.enaml: add missing input` commit where it's difficult to see which input was missing, because the same commit does refactoring.)
* Please comment why a commit gets reverted (in the commit message).

raboof commented:
> * Please keep each commit to the point. (I'm thinking of the `python310Packages.enaml: add missing input` commit where it's difficult to see which input was missing, because the same commit does refactoring.)

Good point, added that review comment to #225972

>     * Please comment why a commit gets reverted (in the commit message).

Good point, updated

bjornfor reviewed (approved):
Thanks!
